test/middleware-test.js

```diff
@@ -1607,6 +1607,71 @@ describe('input-validation middleware tests', function () {
                     done();
                 });
         });
+        it('bad request - wrong content-type (should be application/json)', function (done) {
```

Comment:
I presume that case when swagger specifies no consumed content type at all so it accepts whatever is covered by preexisting tests?

Reply:
Actually, it looks like used swagger defaults to application/json, so this case is most likely not covered. Would be nice to have.